src/utilities/addArticle.js

```diff
@@ -0,0 +1,45 @@
+export function addArticle(
+  event,
+  articleForm,
+  articleURL,
+  allArticles,
+  articleContainer,
+  errorInfo,
+) {
+  event.preventDefault();
+  const titleInput = articleForm.querySelector('.title').value;
+  const contentInput = articleForm.querySelector('.content').value;
+
```

Comment:
Handle the situation where `querySelector` returns `null`.

Reply:
You're still not handling that

Reply:
it should be this way?: const contentInput = articleForm.querySelector('.content')?.value;